### ISSUE TITLE
import/order --fix works something wrong

### ISSUE THREAD

**Issue by sshplendid** [benmosher/eslint-plugin-import#1855]
First of all, I appreciate for developing and maintaining this plugin.🙏

I found malfunctions in `import/order --fix` operation.

I used this plugin and es-lint with `lint-staged`. 
After I committed, I found `import/order` warnings in my IDE. (i set import/order level as `WARN`)
So I did `es-lint --fix` operation again, But that file has still warnings.

That file has almost 200 import lines.

```json
{
    "eslint": "^5.16.0",
    "eslint-plugin-import": "^2.20.2",
}
```

Is it a bug? or Do I need something more?

**Comment by ljharb** [comment]
I'm not sure what you're asking. Not all rules and warnings are autofixable; it's *always* expected that after autofixing, you still have to manually fix some things.

**Comment by sshplendid** [comment]
@ljharb Thanks for comment.
I just talked about [import/order](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md).
Is it possible code to exist not-sorted import statements After `es-lint --fix`?

**Comment by ljharb** [comment]
@sshplendid yes, it may take multiple passes to fully autofix. Is that what you're seeing?
